src/auth/entities/user/ui/LoginField.tsx

```diff
@@ -0,0 +1,23 @@
+import { TextField } from '@/shared/ui/textfield';
+
+interface LoginFieldProps {
+  label: string;
+  placeholder: string;
+  style?: string;
+}
+
+export default function LoginField({
+  label,
+  placeholder,
+  style,
+}: LoginFieldProps) {
+  return (
+    <TextField
+      inputStyle="mt-2 rounded-sm bg-black border placeholder:pl-3 py-2"
```

Comment:
Can't we declare those class names within the `<TextField />` component instead and remove the `inputStyle` prop?